lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss

```diff
@@ -1,5 +1,5 @@
-#instructor-dashboard-content {
-    #extensions {
+#main {
```

Comment:
Do we even need to specify `#main`?
Since we have an ID, shouldn't we just use `#instructor-dashboard-content`?

Reply:
that's precisely what I'm fixing, there's other slick-grids outside of `#instructor-dashboard-content`...